ratis-server/src/main/java/org/apache/ratis/server/impl/ServerImplUtils.java

```diff
@@ -47,14 +50,12 @@ public static RaftServerProxy newRaftServer(
       RaftPeerId id, RaftGroup group, StateMachine.Registry stateMachineRegistry,
       RaftProperties properties, Parameters parameters) throws IOException {
     RaftServer.LOG.debug("newRaftServer: {}, {}", id, group);
-    final RaftServerProxy proxy = newRaftServer(id, stateMachineRegistry, properties, parameters);
-    proxy.initGroups(group);
-    return proxy;
-  }
 
-  private static RaftServerProxy newRaftServer(
-      RaftPeerId id, StateMachine.Registry stateMachineRegistry, RaftProperties properties, Parameters parameters)
-      throws IOException {
```

Comment:
Please revert the unrelated changes.  The original code is more clear.

Reply:
Reverted.